dots.ipynb

```diff
@@ -0,0 +1,637 @@
+{
```

Comment:
<p>To visualize the results of this dot calling, we overlay small rectangles at the positions of the called dots over the HiC map. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>